lib/user-profile-manager/user-profile-manager.js

```diff
@@ -17,19 +17,18 @@ const Q = require("q");
 const _ = require("underscore");
```

Comment:
this file has bad indentation, plz format

Reply:
you didn't take the last version